payment/src/main/java/com/kyriba/conference/payment/service/PaymentMethodService.java

```diff
@@ -0,0 +1,20 @@
+package com.kyriba.conference.payment.service;
+
+import com.kyriba.conference.payment.api.dto.PaymentMethodDto;
+import com.kyriba.conference.payment.api.dto.PaymentMethodResponse;
+import com.kyriba.conference.payment.api.dto.PaymentMethodUrlUpdateDto;
+import com.kyriba.conference.payment.domain.PaymentMethodType;
+
+import java.util.List;
+
+public interface PaymentMethodService {
+    List<PaymentMethodDto> findAll();
```

Comment:
not a good idea to return all payments (I assume we can have lots of them) > try PagingAndSortingRepository

Reply:
I think this is normal for this task.